sys/reaper/reaper_unix.go

```diff
@@ -107,3 +113,21 @@ func (m *Monitor) Unsubscribe(c chan runc.Exit) {
 	close(c)
 	m.Unlock()
 }
+
+func notify(e runc.Exit, subscribers []chan runc.Exit) {
+	for i := 0; i < 50; i++ {
+		var failed []chan runc.Exit
+		for _, s := range subscribers {
+			select {
+			case s <- e:
+			case <-time.After(10 * time.Millisecond):
```

Comment:
Not sure I really care but you `leak` `len(subscribers)` timers for 10ms on the success case. You could just create the timer and call `Stop()` on success to free resources early.

Reply:
ahh, good.  I'll make this change

Reply:
I like it!